tutorials/time-series-modeling/Lab_05.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Time Series Modeling
+description: This lab will go over the fundamentals of Time Series Modeling, including decomposition, autocorrelation and modeling historical changes.
+author: ghidora77, benamie
+tags: remote-sensing, time-series, satellite
+date_published: 2022-02-01
+---
+<!--
+Copyright 2021 The Google Earth Engine Community Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Lab 05 - Time Series Modeling
```

Comment:
Please remove this title. The page title will come from the title in the yaml header.

Reply:
Removed